packages/docz-theme-default/src/index.tsx

```diff
@@ -67,4 +67,6 @@ const enhance = theme(config, ({ mode, codemirrorTheme, ...config }) => ({
   },
 }))
 
+export {components}
```

Comment:
I think it would be also nice to export `enhance` here, because many docz `components` use the context with settings, created there. maybe export `config` and `modes` would be useful too

Reply:
Thanks, could you explain what you mean in code, please?

Reply:
yeah, for example. at first, it's important for the main `Theme` component, like

```js
import {ThemeConfig, DocPreview} from 'docz';
import {components, enhance} from 'docz-theme-default';

const Theme = () => (
    <ThemeConfig>
        {config => (
            <ThemeProvider
                theme={prev => ({
                    ...prev,
                    docz: config.themeConfig,
                })}
            >
                <DocPreview
                    components={{
                        page: Page,
                        render: components.render,
                        inlineCode: components.inlineCode,
                        loading: components.loading,
                        table: components.table,
                        pre: components.pre,
                        tooltip: components.tooltip,
                        ...customComponents,
                    }}
                />
            </ThemeProvider>
        )}
    </ThemeConfig>
);

export default enhance(Theme);
```